source/common/http/conn_manager_impl.h

```diff
@@ -74,7 +74,6 @@ namespace Http {
   COUNTER(downstream_rq_tx_reset)                                                                  \
   COUNTER(downstream_rq_non_relative_path)                                                         \
   COUNTER(downstream_rq_ws_on_non_ws_route)                                                        \
-  COUNTER(downstream_rq_non_ws_on_ws_route)                                                        \
```

Comment:
Is there a doc update needed here? If not, can you add the WS stats to the conn manager docs?